flow/shared-light/mysql/validation.go

```diff
@@ -64,12 +62,12 @@ func CheckMySQL5BinlogSettings(conn *client.Conn, logger log.Logger) error {
 	}
 	row := rs.Values[0]
 
-	binlogFormat := shared.UnsafeFastReadOnlyBytesToString(row[0].AsString())
```

Comment:
these functions get copied around a lot, in go-mysql it's under [zeroalloc](https://github.com/go-mysql-org/go-mysql/blob/master/utils/zeroalloc.go) & every large go package ends up rolling copies of these